### PR TITLE
Add support for SimpleReach measurement

### PR DESCRIPTION
**Pull request by devdazed**
See #1511 for more information.

### CODE REVIEW

**Comment by cramforce** [comment]
Ping for @avimehta @rudygalfi

**Comment by rudygalfi** [comment]
LGTM. @avimehta Can you take a look, too?

File: extensions/amp-analytics/amp-analytics.md [hunk 1]
```diff
@@ -164,6 +164,12 @@ Type attribute value: `quantcast`
 
 Adds support for Quantcast Measurement. More details for adding Quantcast Measurement can be found at [quantcast.com](https://www.quantcast.com/help/guides/)
 
+##### SimpleReach
+
+Type attribute value: `simplereach`
+
+Adds support for SimpleReach.  Configuration details can be found at [simplereach.com/docs](http://docs.simplereach.com/dev-guide/implementation/google-amp-implementation)
```
Comment (rudygalfi) [inline extensions/amp-analytics/amp-analytics.md]: http://docs.simplereach.com/dev-guide/implementation/google-amp-implementation appears to be an empty page. What's your timeline for publishing something there?

**Comment by rudygalfi** [comment]
Just one more comment on the documentation page. After that, I'm willing to LGTM and merge.

File: extensions/amp-analytics/amp-analytics.md (hunk 1 above)
Comment (devdazed) [inline extensions/amp-analytics/amp-analytics.md]: We can deploy that page as early as tomorrow.  You can view the staging version here: http://docs.staging-001.simplereach.com.s3-website-us-east-1.amazonaws.com/dev-guide/implementation/google-amp

Comment (rudygalfi) [inline extensions/amp-analytics/amp-analytics.md]: SGTM, thanks!

**Pull request merged**